src/solvers/flattening/bv_pointers.cpp

```diff
@@ -798,8 +798,8 @@ void bv_pointerst::do_postponed(
         if(size_expr.is_nil())
```

Comment:
A few lines above, the `ns.follow` ought to be eliminated - `size_of_expr` will take care of this (including, once necessary, tag types).

Reply:
Indeed, done